stream_alert_cli/terraform_generate.py

```diff
@@ -360,7 +386,11 @@ def generate_cloudtrail(cluster_name, cluster_dict, config):
         'enable_logging': cloudtrail_enabled,
         'source': 'modules/tf_stream_alert_cloudtrail',
         's3_logging_bucket': '{}.streamalert.s3-logging'.format(
-            config['global']['account']['prefix'])}
+            config['global']['account']['prefix']),
+        'existing_trail': existing_trail,
+        'is_global_trail': is_global_trail,
+        'event_pattern': json.dumps(event_pattern)
```

Comment:
Why do we want `event_pattern` to be a json string within a dictionary as opposed to just another python dict? I assume this has something to do with the config stuff below?